### PR TITLE
fix logging

### PR DESCRIPTION
**Pull request by Talits**

### CODE REVIEW

**Comment by Talits** [comment]
[Garantir integridade na persistencia dos logs](https://trello.com/c/4CfaFTSd/129-garantir-integridade-na-persistencia-dos-logs)

File: ansible/hosts-v3.10.in [hunk 1]
```diff
@@ -1198,6 +1198,7 @@ openshift_enable_unsupported_configurations=false
 
 openshift_disable_check=disk_availability,memory_availability,package_version
 openshift_clusterid="${CLUSTER_ID}"
+openshift_cloudprovider_kind="${PROVIDER}"
```
Comment (caruccio) [inline ansible/hosts-v3.10.in]: This var is already defined in https://github.com/getupcloud/getup-engine-installer/blob/master/bin/gen-hosts#L79

File: ansible/hosts-v3.6.in [hunk 2]
```diff
@@ -913,6 +913,7 @@ etcd_ca_default_days=1825
 ## Extra configs
 openshift_disable_check=disk_availability,memory_availability,package_version
 openshift_clusterid="${CLUSTER_ID}"
+openshift_cloudprovider_kind="${PROVIDER}"
```
Comment (caruccio) [inline ansible/hosts-v3.6.in]: This var is already defined in https://github.com/getupcloud/getup-engine-installer/blob/master/bin/gen-hosts#L79

File: ansible/hosts-v3.7.in [hunk 3]
```diff
@@ -1132,6 +1132,7 @@ openshift_node_open_ports=[{"service":"node-exporter","port":"9100/tcp"},{"servi
 
 openshift_disable_check=disk_availability,memory_availability,package_version
 openshift_clusterid="${CLUSTER_ID}"
+openshift_cloudprovider_kind="${PROVIDER}"
```
Comment (caruccio) [inline ansible/hosts-v3.7.in]: This var is already defined in https://github.com/getupcloud/getup-engine-installer/blob/master/bin/gen-hosts#L79

File: ansible/hosts-v3.9.in [hunk 4]
```diff
@@ -1197,6 +1197,7 @@ openshift_enable_unsupported_configurations=false
 
 openshift_disable_check=disk_availability,memory_availability,package_version
 openshift_clusterid="${CLUSTER_ID}"
+openshift_cloudprovider_kind="${PROVIDER}"
```
Comment (caruccio) [inline ansible/hosts-v3.9.in]: This var is already defined in https://github.com/getupcloud/getup-engine-installer/blob/master/bin/gen-hosts#L79

File: ansible/hosts-v3.7.in (hunk 3 above)
Comment (Talits) [reply ansible/hosts-v3.7.in]: ok

**Pull request merged**